src/platform/forms-system/test/js/fields/FileField.unit.spec.jsx

```diff
@@ -919,4 +920,426 @@ describe('Schemaform <FileField>', () => {
+        ],
+      };
+      const uiSchema = fileUploadUI('Files');
+      const formData = [
+        {
+          errorMessage: 'asdfas',
+        },
+      ];
+      const errorSchema = {
+        0: {
+          __errors: ['Bad error'],
+        },
+      };
+      const registry = {
+        fields: {
+          SchemaField: f => f,
+        },
+      };
+      const tree = shallow(
+        <FileField
+          registry={registry}
+          schema={schema}
+          uiSchema={uiSchema}
+          idSchema={idSchema}
+          errorSchema={errorSchema}
+          formData={formData}
+          formContext={formContext}
+          onChange={f => f}
+          requiredSchema={requiredSchema}
+        />,
+      );
+
+      const cancelButton = tree.find('button.usa-button-secondary');
+      expect(cancelButton.text()).to.equal('Delete file');
+      tree.unmount();
+    });
+
+    it('should render delete button', () => {
+      const idSchema = {
+        $id: 'field',
+      };
+      const schema = {
+        additionalItems: {},
+        items: [
+          {
+            properties: {},
+          },
+        ],
+      };
+      const uiSchema = fileUploadUI('Files');
+      const formData = [
+        {
+          uploading: false,
+        },
+      ];
+      const registry = {
+        fields: {
+          SchemaField: f => f,
+        },
+      };
+      const tree = shallow(
+        <FileField
+          registry={registry}
+          schema={schema}
+          uiSchema={uiSchema}
+          idSchema={idSchema}
+          formData={formData}
+          formContext={formContext}
+          onChange={f => f}
+          requiredSchema={requiredSchema}
+        />,
+      );
+
+      const cancelButton = tree.find('button.usa-button-secondary');
+      expect(cancelButton.text()).to.equal('Delete file');
+      tree.unmount();
+    });
+
+    it('should not render Try again button', () => {
+      const idSchema = {
+        $id: 'field',
+      };
+      const schema = {
+        additionalItems: {},
+        items: [
+          {
+            properties: {},
+          },
+        ],
+      };
+      const uiSchema = fileUploadUI('Files');
+      const formData = [
+        {
+          errorMessage: 'asdfas',
+        },
+      ];
+      const errorSchema = {
+        0: {
+          __errors: [FILE_UPLOAD_NETWORK_ERROR_MESSAGE],
+        },
+      };
+      const registry = {
+        fields: {
+          SchemaField: f => f,
+        },
+      };
+      const tree = shallow(
+        <FileField
+          registry={registry}
+          schema={schema}
+          uiSchema={uiSchema}
+          idSchema={idSchema}
+          errorSchema={errorSchema}
+          formData={formData}
+          formContext={formContext}
+          onChange={f => f}
+          requiredSchema={requiredSchema}
+        />,
+      );
+
+      const retryButton = tree.find('button.usa-button-primary');
+      expect(retryButton.exists()).to.be.false;
```

Comment:
It's not obvious from this that the "retry" button is a primary button - can you make that more clear?

Reply:
Added a comment to help clarify. Let me know if you meant something else.